src/behavioural-events/event.ts

```diff
@@ -7,6 +7,7 @@ export type EventData = {
   appUserId: string;
   properties: EventProperties;
   context: EventContext;
+  workflowIdentifier?: string;
```

Comment:
Correct me if I'm wrong but these events are only currently sent during the paywall/purchase flow correct?